### ISSUE TITLE
how to increase png resolution with altair.renderers.enable("png")

### ISSUE THREAD

**Issue by zyxue** [altair-viz/altair_saver#75]
I'm using altair in a notebook, I tried
```
altair.renderers.enable("png", scale_factor=3.0)
```

it doesn't seem to have any effect on the resolution of the generated png. What's the right way to do so, please?

**Comment by ausiddiqui** [comment]
Facing the same. Installed using the `conda` method on macOS 11.2.3 running python 3.7 and altair 4.10 and vega-lite-cli 5.17.0. I also tried passing `webdriver='firefox' and webdriver='chrome'` and it made not impact. Additionally the axis tick labels are a much lighter style than the level visible on my browser (Safari using Jupyter Lab).

**Comment by sa-tony** [comment]
Try `altair_saver.save(plot, "plot.png", vega_cli_options=["-s 4"])` where 4 is the scaling factor

**Comment by JavierMedel** [comment]
specify the width and height using the properties I hope it helps.

altair_saver.save(bars.properties(width=800, height=400), "plot.png", vega_cli_options=["-s 4"])

**Comment by aecio** [comment]
This seems to work when using notebooks without having to use `save()` function:

```
alt.renderers.enable('altair_saver', fmts=['png'], embed_options={'scaleFactor': '4'})
```

**Comment by sid-kap** [comment]
I'm trying to install altair_saver, but both `pip install altair-saver` and `pip install git+https://github.com/altair-viz/altair_saver/` fail (the latter says it's only available in python 2.7). Am I doing something wrong?

**Comment by joelostblom** [comment]
Since Altair 5.2, the functionality of Altair Saver is now available in Altair via the [vl-convert](https://github.com/vega/vl-convert) package. Most of the functionality has been available since 5.0, and [the main addition in 5.2 was PDF export](https://github.com/altair-viz/altair/pull/3244). See [the docs on how to save charts](https://altair-viz.github.io/user_guide/saving_charts.html#png-svg-and-pdf-format) for more details. This includes how to set the resolution https://altair-viz.github.io/user_guide/saving_charts.html#png-figure-size-resolution

We are going to archive this repo, so I'm closing all the open issues and PRs before doing so. Try out the new options for saving charts mentioned above and if you run into issues, please open an issue directly in the altair or vl-convert repo.
